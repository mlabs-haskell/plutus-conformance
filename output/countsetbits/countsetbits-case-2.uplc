[ (builtin countSetBits) (con bytestring #0000) ]