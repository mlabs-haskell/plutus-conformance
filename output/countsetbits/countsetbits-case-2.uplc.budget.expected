{"exBudgetCPU":100000048100,"exBudgetMemory":100000000400}